src/components/EpisodeContainer.tsx

```diff
@@ -0,0 +1,31 @@
+// utils
+import { filterInput } from "../utils/filterinput";
+import { IEpisode } from "../utils/Interfaces";
+// components
+import Episode from "./Episode";
+
+interface EpisodeContainerProps {
+    episodesToDisplay: IEpisode[];
+    searchInput: string;
+}
+
+export default function EpisodeContainer({
+    episodesToDisplay,
+    searchInput,
+}: EpisodeContainerProps): JSX.Element {
+    const currentEpisodes = episodesToDisplay.filter((episode) =>
+        filterInput(episode.name, episode.summary, searchInput)
+    );
+
+    const allEpisodes = currentEpisodes.map((episode) => {
+        return <Episode {...episode} key={episode.id} />;
+    });
```

Comment:
Great idea to have a Container component for mapping through episodes inside and no directly in the App component.